cves/CVE-2013-2115.yml

```diff
@@ -45,7 +46,14 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+
+description: |
+  There are tags in Struts (s:url, s:a) that accept an attribute called
+  includeParams. It only accepts three values: none, get, and all. If
+  a user creates a request using Object-Graph Navigation Language (OGNL)
+  in it, this will create a bad request to the server, where a request
+  may be able to manipulate context objects with different privileges
+  than they should have. This was also caused by a fix for CVE-2013-1966.
```

Comment:
+3 on being related to another similar cve not being completely fixed. Super cool.